### PR TITLE
chore: apply prettier to more files

### PR DESCRIPTION
**Pull request by benjie**
Uses `.eslintignore` as if it were `.prettierignore` so we can now apply `prettier` to everything that ESLint should apply to (AND MORE!)

Note: I dropped `toml` from the list of prettier extensions since it throws an error when it matches due to missing plugin, and the plugin is alpha. The toml file didn't match the previous glob.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ The purpose of this monorepo is to give the GraphQL Community:
 
 ## [GraphiQL](packages/graphiql#readme)
 
+<!-- prettier-ignore -->
```
Comment (benjie) [inline README.md]: This is the only manual change in this commit.

File: .eslintignore [hunk 2]
```diff
@@ -48,3 +49,8 @@ packages/graphiql/lsp
 
 # Vendored files
 /packages/graphiql/test/vendor
+
+
+# Autogenerated CHANGELOGs
```
Comment (acao) [inline .eslintignore]: lets just make it all MDs, as we also have the ones at the base?

Comment (benjie) [reply .eslintignore]: We'll have to make it run pretty after creating these then, but I'm happy with that.

**Comment by benjie** [comment]
These are the files being prettified now:

```
.eslintrc.js 61ms
babel.config.js 10ms
CHANGELOG.md 53ms
commitlint.config.js 4ms
CONTRIBUTING.md 17ms
cypress.json 4ms
DEVELOPMENT.md 20ms
examples/graphiql-cdn/CHANGELOG.md 6ms
examples/graphiql-cdn/index.html 55ms
examples/graphiql-cdn/package.json 5ms
examples/graphiql-cdn/README.md 4ms
examples/graphiql-webpack/babel.config.js 6ms
examples/graphiql-webpack/CHANGELOG.md 10ms
examples/graphiql-webpack/package.json 4ms
examples/graphiql-webpack/README.md 7ms
examples/graphiql-webpack/src/index.jsx 10ms
examples/graphiql-webpack/webpack.config.js 11ms
jest.config.js 9ms
lerna.json 5ms
package.json 8ms
packages/codemirror-graphql/__tests__/hint-test.js 80ms
packages/codemirror-graphql/__tests__/lint-test.js 15ms
packages/codemirror-graphql/__tests__/mode-test.js 18ms
packages/codemirror-graphql/__tests__/testSchema.js 17ms
packages/codemirror-graphql/babel.config.js 4ms
packages/codemirror-graphql/CHANGELOG.md 16ms
packages/codemirror-graphql/package.json 5ms
packages/codemirror-graphql/README.md 9ms
packages/codemirror-graphql/resources/bootJSDOM.js 9ms
packages/codemirror-graphql/resources/mochaBootload.js 9ms
packages/codemirror-graphql/resources/watch.js 25ms
packages/codemirror-graphql/src/__tests__/hint-test.js 30ms
packages/codemirror-graphql/src/__tests__/lint-test.js 8ms
packages/codemirror-graphql/src/__tests__/mode-test.js 15ms
packages/codemirror-graphql/src/__tests__/testSchema.js 21ms
packages/codemirror-graphql/src/hint.js 10ms
packages/codemirror-graphql/src/info.js 17ms
packages/codemirror-graphql/src/jump.js 7ms
packages/codemirror-graphql/src/lint.js 5ms
packages/codemirror-graphql/src/mode.js 6ms
packages/codemirror-graphql/src/results/__tests__/mode-test.js 6ms
packages/codemirror-graphql/src/results/mode.js 8ms
packages/codemirror-graphql/src/utils/__tests__/jsonParse-test.js 6ms
packages/codemirror-graphql/src/utils/collectVariables.js 5ms
packages/codemirror-graphql/src/utils/forEachState.js 4ms
packages/codemirror-graphql/src/utils/getTypeInfo.js 10ms
packages/codemirror-graphql/src/utils/hintList.js 13ms
packages/codemirror-graphql/src/utils/info-addon.js 14ms
packages/codemirror-graphql/src/utils/jsonParse.js 18ms
packages/codemirror-graphql/src/utils/jump-addon.js 11ms
packages/codemirror-graphql/src/utils/runParser.js 4ms
packages/codemirror-graphql/src/utils/SchemaReference.js 11ms
packages/codemirror-graphql/src/variables/__tests__/hint-test.js 23ms
packages/codemirror-graphql/src/variables/__tests__/lint-test.js 10ms
packages/codemirror-graphql/src/variables/__tests__/mode-test.js 11ms
packages/codemirror-graphql/src/variables/hint.js 18ms
packages/codemirror-graphql/src/variables/lint.js 15ms
packages/codemirror-graphql/src/variables/mode.js 12ms
packages/graphiql/__mocks__/codemirror.js 7ms
packages/graphiql/.babelrc.js 3ms
packages/graphiql/.eslintrc.js 18ms
packages/graphiql/.storybook/config.js 5ms
packages/graphiql/babel.config.js 3ms
packages/graphiql/CHANGELOG.md 23ms
packages/graphiql/cypress.json 3ms
packages/graphiql/cypress/fixtures/example.json 3ms
packages/graphiql/cypress/integration/init.spec.js 6ms
packages/graphiql/cypress/integration/prettify.spec.js 10ms
packages/graphiql/cypress/plugins/index.js 4ms
packages/graphiql/cypress/support/commands.js 6ms
packages/graphiql/cypress/support/index.js 3ms
packages/graphiql/package.json 5ms
packages/graphiql/postcss.config.js 4ms
packages/graphiql/README.md 98ms
packages/graphiql/resources/webpack.config.js 14ms
packages/graphiql/src/cdn.js 6ms
packages/graphiql/src/components/__tests__/DocExplorer.spec.js 6ms
packages/graphiql/src/components/__tests__/ExampleSchema.js 7ms
packages/graphiql/src/components/__tests__/fixtures.js 6ms
packages/graphiql/src/components/__tests__/GraphiQL.spec.js 40ms
packages/graphiql/src/components/__tests__/helpers/codeMirror.js 4ms
packages/graphiql/src/components/__tests__/helpers/storage.js 4ms
packages/graphiql/src/components/__tests__/HistoryQuery.spec.js 7ms
packages/graphiql/src/components/DocExplorer.js 13ms
packages/graphiql/src/components/DocExplorer/__tests__/FieldDoc.spec.js 8ms
packages/graphiql/src/components/DocExplorer/__tests__/TypeDoc.spec.js 11ms
packages/graphiql/src/components/DocExplorer/__tests__/TypeLink.spec.js 8ms
packages/graphiql/src/components/DocExplorer/Argument.js 6ms
packages/graphiql/src/components/DocExplorer/DefaultValue.js 4ms
packages/graphiql/src/components/DocExplorer/FieldDoc.js 6ms
packages/graphiql/src/components/DocExplorer/MarkdownContent.js 4ms
packages/graphiql/src/components/DocExplorer/SchemaDoc.js 7ms
packages/graphiql/src/components/DocExplorer/SearchBox.js 6ms
packages/graphiql/src/components/DocExplorer/SearchResults.js 12ms
packages/graphiql/src/components/DocExplorer/TypeDoc.js 14ms
packages/graphiql/src/components/DocExplorer/TypeLink.js 5ms
packages/graphiql/src/components/ExecuteButton.js 12ms
packages/graphiql/src/components/GraphiQL.js 66ms
packages/graphiql/src/components/HistoryQuery.js 11ms
packages/graphiql/src/components/ImagePreview.js 8ms
packages/graphiql/src/components/QueryEditor.js 14ms
packages/graphiql/src/components/QueryHistory.js 10ms
packages/graphiql/src/components/ResultViewer.js 8ms
packages/graphiql/src/components/ToolbarButton.js 5ms
packages/graphiql/src/components/ToolbarGroup.js 3ms
packages/graphiql/src/components/ToolbarMenu.js 7ms
packages/graphiql/src/components/ToolbarSelect.js 8ms
packages/graphiql/src/components/VariableEditor.js 10ms
packages/graphiql/src/index.js 3ms
packages/graphiql/src/new-components/__tests__/stories.spec.js 3ms
packages/graphiql/src/new-components/Layout.stories.js 6ms
packages/graphiql/src/new-components/List.js 6ms
packages/graphiql/src/new-components/List.stories.js 6ms
packages/graphiql/src/new-components/Nav.js 5ms
packages/graphiql/src/new-components/Tabs.js 9ms
packages/graphiql/src/new-components/Tabs.stories.js 8ms
packages/graphiql/src/new-components/themes/constants.js 4ms
packages/graphiql/src/new-components/themes/default/index.js 5ms
packages/graphiql/src/new-components/themes/default/Layout.js 8ms
packages/graphiql/src/new-components/themes/provider.js 5ms
packages/graphiql/src/new-components/themes/README.md 11ms
packages/graphiql/src/new-components/Type.js 4ms
packages/graphiql/src/new-components/Type.stories.js 4ms
packages/graphiql/src/utility/__tests__/find.spec.js 5ms
packages/graphiql/src/utility/__tests__/getQueryFacts.spec.js 7ms
packages/graphiql/src/utility/__tests__/mergeAst-fixture.js 5ms
packages/graphiql/src/utility/__tests__/mergeAst.spec.js 7ms
packages/graphiql/src/utility/__tests__/normalizeWhitespace.spec.js 4ms
packages/graphiql/src/utility/__tests__/QueryStore.spec.js 13ms
packages/graphiql/src/utility/__tests__/StorageAPI.spec.js 11ms
packages/graphiql/src/utility/CodeMirrorSizer.js 4ms
packages/graphiql/src/utility/commonKeys.js 3ms
packages/graphiql/src/utility/debounce.js 4ms
packages/graphiql/src/utility/elementPosition.js 3ms
packages/graphiql/src/utility/fillLeafs.js 9ms
packages/graphiql/src/utility/find.js 4ms
packages/graphiql/src/utility/getQueryFacts.js 5ms
packages/graphiql/src/utility/getSelectedOperationName.js 5ms
packages/graphiql/src/utility/introspectionQueries.js 4ms
packages/graphiql/src/utility/mergeAst.js 6ms
packages/graphiql/src/utility/normalizeWhitespace.js 4ms
packages/graphiql/src/utility/onHasCompletion.js 7ms
packages/graphiql/src/utility/QueryStore.js 10ms
packages/graphiql/src/utility/StorageAPI.js 6ms
packages/graphiql/test/beforeDevServer.js 4ms
packages/graphiql/test/e2e-server.js 6ms
packages/graphiql/test/README.md 5ms
packages/graphiql/test/schema.js 14ms
packages/graphql-language-service-interface/CHANGELOG.md 11ms
packages/graphql-language-service-interface/package.json 3ms
packages/graphql-language-service-interface/README.md 5ms
packages/graphql-language-service-interface/src/__tests__/getAutocompleteSuggestions-test.ts 141ms
packages/graphql-language-service-interface/src/__tests__/getDefinition-test.ts 11ms
packages/graphql-language-service-interface/src/__tests__/getDiagnostics-test.ts 14ms
packages/graphql-language-service-interface/src/__tests__/getHoverInformation-test.ts 16ms
packages/graphql-language-service-interface/src/__tests__/getOutline-test.ts 11ms
packages/graphql-language-service-interface/src/__tests__/GraphQLLanguageService-test.ts 12ms
packages/graphql-language-service-interface/src/autocompleteUtils.ts 25ms
packages/graphql-language-service-interface/src/getAutocompleteSuggestions.ts 65ms
packages/graphql-language-service-interface/src/getDefinition.ts 12ms
packages/graphql-language-service-interface/src/getDiagnostics.ts 16ms
packages/graphql-language-service-interface/src/getHoverInformation.ts 18ms
packages/graphql-language-service-interface/src/getOutline.ts 13ms
packages/graphql-language-service-interface/src/GraphQLLanguageService.ts 22ms
packages/graphql-language-service-interface/src/index.ts 4ms
packages/graphql-language-service-interface/tsconfig.esm.json 4ms
packages/graphql-language-service-interface/tsconfig.json 4ms
packages/graphql-language-service-parser/CHANGELOG.md 9ms
packages/graphql-language-service-parser/package.json 5ms
packages/graphql-language-service-parser/README.md 6ms
packages/graphql-language-service-parser/src/CharacterStream.ts 14ms
packages/graphql-language-service-parser/src/index.ts 4ms
packages/graphql-language-service-parser/src/onlineParser.ts 24ms
packages/graphql-language-service-parser/src/RuleHelpers.ts 6ms
packages/graphql-language-service-parser/src/Rules.ts 23ms
packages/graphql-language-service-parser/tsconfig.esm.json 3ms
packages/graphql-language-service-parser/tsconfig.json 3ms
packages/graphql-language-service-server/CHANGELOG.md 12ms
packages/graphql-language-service-server/package.json 4ms
packages/graphql-language-service-server/README.md 4ms
packages/graphql-language-service-server/src/__tests__/GraphQLCache-test.js 19ms
packages/graphql-language-service-server/src/__tests__/MessageProcessor-test.js 12ms
packages/graphql-language-service-server/src/findGraphQLTags.js 14ms
packages/graphql-language-service-server/src/GraphQLCache.js 49ms
packages/graphql-language-service-server/src/GraphQLWatchman.js 9ms
packages/graphql-language-service-server/src/index.js 4ms
packages/graphql-language-service-server/src/Logger.js 6ms
packages/graphql-language-service-server/src/MessageProcessor.js 33ms
packages/graphql-language-service-server/src/startServer.js 10ms
packages/graphql-language-service-server/src/stringToHash.js 3ms
packages/graphql-language-service-types/CHANGELOG.md 13ms
packages/graphql-language-service-types/package.json 3ms
packages/graphql-language-service-types/README.md 4ms
packages/graphql-language-service-types/src/index.ts 27ms
packages/graphql-language-service-types/tsconfig.esm.json 3ms
packages/graphql-language-service-types/tsconfig.json 3ms
packages/graphql-language-service-utils/CHANGELOG.md 12ms
packages/graphql-language-service-utils/package.json 4ms
packages/graphql-language-service-utils/README.md 3ms
packages/graphql-language-service-utils/src/__tests__/__fixtures__/file.js 3ms
packages/graphql-language-service-utils/src/__tests__/__fixtures__/package.json 3ms
packages/graphql-language-service-utils/src/__tests__/file-test.ts 13ms
packages/graphql-language-service-utils/src/__tests__/getASTNodeAtPosition-test.ts 7ms
packages/graphql-language-service-utils/src/__tests__/Range-test.ts 10ms
packages/graphql-language-service-utils/src/__tests__/validateWithCustomRules-test.ts 8ms
packages/graphql-language-service-utils/src/file.ts 7ms
packages/graphql-language-service-utils/src/getASTNodeAtPosition.ts 5ms
packages/graphql-language-service-utils/src/index.ts 3ms
packages/graphql-language-service-utils/src/Range.ts 8ms
packages/graphql-language-service-utils/src/validateWithCustomRules.ts 8ms
packages/graphql-language-service-utils/tsconfig.esm.json 3ms
packages/graphql-language-service-utils/tsconfig.json 4ms
packages/graphql-language-service/bin/graphql.js 3ms
packages/graphql-language-service/CHANGELOG.md 22ms
packages/graphql-language-service/package.json 3ms
packages/graphql-language-service/README.md 17ms
packages/graphql-language-service/src/__tests__/index-test.js 4ms
packages/graphql-language-service/src/cli.js 8ms
packages/graphql-language-service/src/client.js 11ms
README.md 25ms
RELEASING.md 12ms
resources/buildFlow.js 5ms
resources/buildJs.js 4ms
resources/jestBabelTransform.js 3ms
resources/pretty.js 5ms
resources/README.md 6ms
resources/renameFileExtensions.js 6ms
resources/runTests.js 4ms
resources/test.config.js 6ms
resources/tsconfig.base.cjs.json 6ms
resources/tsconfig.base.esm.json 3ms
resources/tsconfig.build.cjs.json 3ms
resources/tsconfig.build.esm.json 3ms
resources/util.js 4ms
tsconfig.json 3ms
typedoc.json 3ms
wg-agendas/2019-05-14.md 16ms
wg-agendas/2019-06-18.md 11ms
wg-agendas/2019-08-21.md 8ms
```

**Comment by benjie** [comment]
@acao Changelogs now prettier; are you sure this won't break the auto-update of them? Do they get re-written each time or some how diffed?

File: .eslintignore (hunk 2 above)
Comment (acao) [reply .eslintignore]: this is a new addition in package.json:

```json
{
 "lint-staged": {
    "*.{js,ts,jsx,tsx}": [
      "eslint --fix",
      "prettier --write",
      "git add"
    ],
    "*.{md}": [
      "prettier --write",
      "git add"
    ]
  },
}
```

Comment (acao) [reply .eslintignore]: we also perform a prettier check in CI

**Comment by acao** [comment]
@benjie this is actually desired and works great. it just appends the files. so it would be nice to format after every changelog generation, which we do by running `lerna publish`, which runs lerna version and thus git add/commit under the hood

File: resources/pretty.js [hunk 3]
```diff
@@ -14,16 +14,20 @@ const INVERSE = '\x1b[7m';
 const RESET = '\x1b[0m';
 const YELLOW = '\x1b[33m';
 
-const glob =
-  '{packages/*/{resources,src},resources,src}/**/*.{js,ts,jsx,tsx,md,json5,toml,json}';
+// Simple glob because we use .eslintignore to limit it
```
Comment (acao) [inline resources/pretty.js]: 👍

**Pull request merged**